samples/matter/thermostat/README.rst

```diff
@@ -87,8 +87,8 @@ External sensor integration
 The thermsotat sample lets you connect to an external temperature sensor, for example :ref:`Matter weather station application on Nordic Thingy:53 <matter_weather_station_app>`.
 This establishes the :ref:`matter_thermostat_sample_binding` to Matter's temperature measurement cluster.
 
-To enable this feature, set :kconfig:option:`CONFIG_THERMOSTAT_EXTERNAL_SENSOR` to ``y`` in the configuration files or use the ``-DCONFIG_THERMOSTAT_EXTERNAL_SENSOR=y`` :ref:`CMake option <cmake_options>` when building the sample.
-You can then follow steps listed in :ref:`matter_thermostat_sensor_testing`.
+By default thermostat measure internal sensor value if available (in this sample internal temperature is mocked), when binding is established with external sensor thermostat automatically change measurement for bound device.
+To test this feature you can follow steps listed in :ref:`matter_thermostat_sensor_testing`.
```

Comment:
```suggestion
By default, the thermostat sample generates simulated temperature measurements to simulate local temperature changes.
Additionally, you can enable periodic outdoor temperature measurements by binding the thermostat with an external temperature sensor device.
To test this feature you can follow steps listed in :ref:`matter_thermostat_sensor_testing`.
```
@wiba-nordic has the last word on that though ;)

Reply:
```suggestion
By default, the thermostat sample generates simulated temperature measurements that simulate local temperature changes.
Additionally, you can enable periodic outdoor temperature measurements by binding the thermostat with an external temperature sensor device.
To test this feature, follow the steps listed in the :ref:`matter_thermostat_sensor_testing` section.
```
Very good first suggestion, with just a couple of small changes from me.